src/oas3/accessors.ts

```diff
@@ -45,11 +45,14 @@ export function getSecurities(
           ];
         }
 
+        const extensions = scopes?.length ? { ['x-scopes']: scopes } : {};
+
         return [
           opScheme,
           {
             ...definition,
-            extensions: getExtensions(definition),
+            ...extensions,
```

Comment:
Do we need all extensions to be listed here? Can't we rely just on `extensions` object?

Reply:
That's what I thought at the beginning, but it turned out the `extensions` object is completely ignored in the `securities.ts#translateToSingleSecurity` method and calculated again from the `securityScheme` object: 
```ts
  ...
  const baseObject: { id: string; key: string; description?: string; extensions: Extensions } = {
    id: this.generateId.httpSecurity({
      keyOrName: key,
      kind,
      ...(kind === 'requirement'
        ? {
            index,
            scopeKeys: getScopeKeys(securityScheme),
          }
        : {}),
    }),
    key,
    extensions: getExtensions(securityScheme),
  };
  ...
```
We could simply do `extensions: securityScheme.extensions`, but I wasn't sure where else is the method used and if it breaks something else.

Reply:
Yea, you're right about that, we're loosing these extensions. Maybe we could try to include it in there with `extensions: getExtensions({ ...securityScheme, ...securityScheme.extensions })`, so we keep both, but I'm ok with your current approach too.